common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -31,13 +31,7 @@
 import java.text.NumberFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.Enumeration;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Using the '.*' form of import should be avoided

Reply:
Hi，RongtongJin, I've fixed the import issue, please review again, thanks.

Reply:
Btw, why can't I find the method org.apache.rocketmq.tools.admin.DefaultMQAdminExt#examineBrokerClusterAclConfig in the latest version? Does it migrated to other name?